src/config/dynamic-theme-fixes.config

```diff
@@ -3274,6 +3274,20 @@ CSS
 
 ================================
 
+feedly.com
+
+CSS
+.entry.u0:hover {
```

Comment:
Making this selector using the same fix as the fix under this selector a little improvement can be made and moving this selector to the other selectors.

Reply:
You're absolutely right. How about this?

Please also read my changes to my answers above, I found out why it shows `.u4` for you.

Reply:
> Please also read my changes to my answers above, I found out why it shows `.u4` for you.

I've read it and the changes now looks 100% to me and ready to be merged.

Reply:
Awesome! Thanks for your help man!